modin/pandas/test/dataframe/test_default.py

```diff
@@ -615,117 +545,107 @@ def test_replace():
 
 @pytest.mark.parametrize("rule", ["5T", pandas.offsets.Hour()])
 @pytest.mark.parametrize("axis", [0, "columns"])
-@pytest.mark.parametrize("closed", ["left", "right"])
-@pytest.mark.parametrize("label", ["right", "left"])
-@pytest.mark.parametrize("on", [None, "DateColumn"])
-@pytest.mark.parametrize("level", [None, 1])
-def test_resample(rule, axis, closed, label, on, level):
-    freq = "H"
-    base = 2
-    index = pandas.date_range("31/12/2000", periods=12, freq=freq)
-    data = {"A": range(12), "B": range(12)}
-
-    pandas_df = pandas.DataFrame(data, index=index)
+def test_resample(rule, axis):
+    data, index, = (
+        test_data_resample["data"],
+        test_data_resample["index"],
+    )
     modin_df = pd.DataFrame(data, index=index)
-
-    if on is not None and axis == 0:
-        pandas_df[on] = pandas.date_range("22/06/1941", periods=12, freq="T")
-        modin_df[on] = pandas.date_range("22/06/1941", periods=12, freq="T")
-    else:
-        on = None
+    pandas_df = pandas.DataFrame(data, index=index)
 
     if axis == "columns":
         pandas_df = pandas_df.T
         modin_df = modin_df.T
 
-    if level is not None and axis == 0 and on is None:
+    pandas_resampler = pandas_df.resample(
+        rule,
+        axis=axis,
+        base=2,
+    )
+    modin_resampler = modin_df.resample(
+        rule,
+        axis=axis,
+        base=2,
+    )
+
+    for group in [
+        ("count", "sum", "std", "sem", "size", "prod", "ohlc", "quantile"),
+        ("min", "median", "mean", "max", "last", "first", "nunique", "var"),
+        ("interpolate", "asfreq", "pad", "nearest", "bfill", "backfill", "ffill"),
+    ]:
+        for method in group:
+            eval_general(
+                modin_resampler, pandas_resampler, lambda df: getattr(df, method)()
+            )
+
+    for method_arg in [
+        ("pipe", lambda x: x.max() - x.min()),
+        ("transform", lambda x: (x - x.mean()) / x.std()),
+        ("apply", ["sum", "mean", "max"]),
+        ("aggregate", ["sum", "mean", "max"]),
+    ]:
+        method, arg = method_arg[0], method_arg[1]
+        eval_general(
+            modin_resampler, pandas_resampler, lambda df: getattr(df, method)(arg)
+        )
```

Comment:
can we somehow get rid of using for loop on testing parameters by moving them to `parametrize`? It would be some challenging to detect with which params exactly test failed if it happens in a loop.

Reply:
Loops were be removed. Please check.